packages/styled-system/package.json

```diff
@@ -34,7 +40,8 @@
         "react-dom": "*"
     },
     "dependencies": {
-        "react-aria": "3.29.1"
+        "react-aria": "3.29.1",
+        "clsx": "2.0.0"
```

Comment:
The addition of this library is due to one of the previous comments 👍🏻 on the documentation, we use [classnames](https://www.npmjs.com/package/classnames) in other projects.